src/omnisharp/launcher.ts

```diff
@@ -228,6 +228,7 @@ async function launch(cwd: string, args: string[], launchInfo: LaunchInfo, platf
         args.push(`formattingOptions:useTabs=${!getConfigurationValue(globalConfig, csharpConfig, 'editor.insertSpaces', true)}`);
         args.push(`formattingOptions:tabSize=${getConfigurationValue(globalConfig, csharpConfig, 'editor.tabSize', 4)}`);
         args.push(`formattingOptions:indentationSize=${getConfigurationValue(globalConfig, csharpConfig, 'editor.tabSize', 4)}`);
+        args.push('-z')
```

Comment:
I don't think you want this. If you launch OmniSharp with `--zero-based-indeces`, you'll need to update all of the places in C# for VS Code that expect 1-based indeces today.